# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hostap-utils/install.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- hostap-utils-0.4.7/Makefile.vanilla	2007-03-23 15:07:24.000000000 +0100
+++ hostap-utils-0.4.7/Makefile	2007-03-23 15:08:54.000000000 +0100
@@ -2,6 +2,8 @@
 CFLAGS = -O2 -Wall
 endif
 
+prefix=/usr/local
+
 # Include directory for CVS version
 CFLAGS += -I../driver/modules
 
@@ -28,6 +30,9 @@
 
 hostap_rid.o: hostap_rid.c util.h
 
+install:
+	for i in $(all); do cp $$i $(DESTDIR)$(prefix)/bin/$$i; done
+
 clean:
 	rm -f prism2_srec hostap_crypt_conf hostap_diag hostap_rid \
 		hostap_io_debug *.o
